recipe/meta.yaml

```diff
@@ -1,41 +1,50 @@
 {% set name = "jupyter_ydoc" %}
-{% set version = "0.3.3" %}
+{% set version = "0.2.4" %}
+{% set sha256 = "a3f670a69135e90493ffb91d6788efe2632bf42c6cc42a25f25c2e6eddd55a0e" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 9ba0975f2016e0c94e3200aeb713032ae255ad8ad035905bd0574cae42240b73
+  sha256: {{ sha256 }}
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation . -vv
+  # y-py not available for s390x
+  skip: true # [ s390x or py<37 ] 
 
 requirements:
   host:
-    - pip
-    - hatchling
     - hatch-nodejs-version
-    - python >=3.7
+    - hatchling >= 1.10.0
+    - pip
+    - python
+    - wheel
   run:
-    - python >=3.7
+    - python
     - importlib_metadata >=3.6
```

Comment:
Should be `importlib_metadata >=3.6  # [py<310]`, see https://github.com/jupyter-server/jupyter_ydoc/blob/v0.2.4/pyproject.toml